grant/projects/evolutionland.md

```diff
@@ -113,26 +110,21 @@ The second payment is paid in USDT to the value of $3,000 USD on completion of t
 
 The 2nd payment date: When the beta version has passed the inspection.
 
-## **Additional Information**
+## Additional Information
 
 To trial our pre-beta product please go to: Evolution Land. You will need the following:
 
 - Metamask or EVM related wallet
 - Crab for gas and xRING for purchase
 
-
-### **What work has been done so far?**
+### What work has been done so far?
 
 Evolution Land's minimum viable Product has been made and is nearing its beta testing phase which is to be tested by the Darwinia community In the end of Feb 2022.
 
-### **Are there any teams who have already contributed (financially) to the project?**
-
-No.
```

Comment:
Informations in archived grants *SHOULD NOT* be modified, they have archive values similar to MOU and contracts.

Don't obsesses  over formats here, we can spend our time on more valuable things.

Reply:
Okay, I'll revert these changes.